packages/account/src/Constants/routes-config.ts

```diff
@@ -20,13 +20,15 @@ import {
     AccountClosed,
     DeactivateAccount,
     LanguageSettings,
-} from 'Sections';
+} from '../Sections';
```

Comment:
why is this changed to relative path?

Reply:
It was giving errors in console but with relative path issue was resolved.